tests/testthat/test-plot_type1error.R

```diff
@@ -0,0 +1,48 @@
+test_that("Ensure errors are correct", {
+
+  dt1 <- tibble(
+    HR = c(0.8, 0.8),
+    driftHR = c(1,1),
+    prior = c('full_ext','gamma'),
+    pred = c('all','all'),
+    reject = c(0.2, 0.3)
+  )
+
+  dt2 <- tibble(
+    HR = c(0.8, 0.8, 1.0, 1.0),
+    driftHR = c(1,1,1,1),
+    prior = c('full_ext','gamma','full_ext','gamma'),
+    pred = c('all','all','all','all'),
+    reject = c(0.2, 0.3,0.4,0.5)
+  )
+
+  expect_error(plot_type1error(dt1, driftHR=1,pred='all'))
+  expect_error(plot_type1error(dt2, driftHR=1.3,pred='all'))
+  expect_error(plot_type1error(dt2, driftHR=1,pred='cov1'))
```

Comment:
General comment, never use expect_error without at least specifying part of the error message. The reason being that you may end up catching error messages that you didn't intend to catch meaning genuine errors with the code go undetected.

Reply:
@gowerc , great suggestion, thanks.